content/actions/learn-github-actions/events-that-trigger-workflows.md

```diff
@@ -97,6 +97,13 @@ You can manually trigger workflow runs. To trigger specific workflows in a repos
 | ------------------ | ------------ | ------------ | ------------------|
 | [workflow_dispatch](/webhooks/event-payloads/#workflow_dispatch) | n/a | Last commit on the `GITHUB_REF` branch | Branch that received dispatch |
 
+
+{% note %}
+
+**Note:** This manual event can only run a workflow if the workflow file is on the default branch.
+
+{% endnote %}
```

Comment:
We have a "reusable" for this text so that we can keep it consistent across all the different events.

```suggestion
{% data reusables.github-actions.branch-requirement %}
```